policyengine_us/parameters/gov/states/mi/tax/income/exemptions/disabled_veteran.yaml

```diff
@@ -0,0 +1,15 @@
+description: Michigan provides this exemption amount for each disabled veteran in the household.
+values:
+  2017-01-01: 400
```

Comment:
interesting, it's 250 in the law as of 2008, but it hasn't changed from 400 since 2017?

Reply:
doesn't know about the years before 2017, but it was 400 after 2017 in [2021 Michigan Taxpayer Assistance Manual](https://www.michigan.gov/-/media/Project/Websites/taxes/2022RM/UNCAT/2019_Taxpayer_Assistance_Manual.pdf?rev=3a1167cc5dcf4340a4b1e1a8fe76078c)